src/ui/map.ts

```
@@ -79,16 +82,28 @@ class MapUI {
     link.appendChild(img)
     map.getContainer().appendChild(link);
 
+    // add attribution
+    const attribution = new maplibregl.AttributionControl({ compact: false });
```

can we use the default resizing/hiding attribution?